tembo-operator/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "controller"
 description = "Tembo Operator for Postgres"
-version = "0.44.1"
+version = "0.44.2"
```

Comment:
No need to bump the operator version on this. Just bump version on tembo-stacks/Cargo.toml

Reply:
reverted the increment. have a look